src/main/java/de/tum/in/www1/artemis/web/rest/util/StringUtil.java

```diff
@@ -18,4 +18,14 @@ public class StringUtil {
     public static String stripIllegalCharacters(String input) {
         return StringUtils.replaceChars(input, ILLEGAL_CHARACTERS, null);
     }
+
+    /**
+     * Replaces whitespace with underscores and removes illegal characters, which allows to use the string in file names
+     *
+     * @param input String to sanitize
+     * @return sanitized string
+     */
+    public static String sanitizeStringForFileName(String input) {
+        return input.replaceAll("\\s+", "_").replaceAll("[\\\\/:*?#+%$§\"<>|]", "");
```

Comment:
oh okay there it is again. I thought you used a java class

Reply:
so, you would prefer a library method call?

Reply:
so far I haven't found anything that looked like a good fit here.

Reply:
no it's fine as long as the complexity is hidden in a method. I just thought initially that you found a native method doing what we wanted, which would be a perfect solution imo